### PR TITLE
Internationalization Considerations suggested editorial updates

### PR DESCRIPTION
**Pull request by grantnoble**
Wording changes, some grammar, some formatting changes

***
<a href="https://pr-preview.s3.amazonaws.com/grantnoble/vc-data-model/pull/692.html" title="Last updated on Jul 15, 2019, 6:28 PM UTC (f17c59e)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/692/96dc09e...grantnoble:f17c59e.html" title="Last updated on Jul 15, 2019, 6:28 PM UTC (f17c59e)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4802,23 +4802,23 @@ <h3>Data First Approaches</h3>
     <h2>Internationalization Considerations</h2>
 
     <p>
-There are a number of internationalization considerations implementers
-are advised to be aware of when publishing data described in this specification.
+There are a number of internationalization considerations implementers are
+advised to be aware of when publishing data described in this specification.
```
Comment (TallTed) [inline index.html]:
```
There are a number of internationalization considerations implementers are
advised to be aware of when publishing data described in this specification.
```
->
```
Implementers are advised to be aware of a number of internationalization
considerations when publishing data described in this specification.
```

File: index.html [hunk 2]
```diff
@@ -4802,23 +4802,23 @@ <h3>Data First Approaches</h3>
...
 As with any web standards or protocols implementation, ignoring
 internationalization makes it difficult for data to be produced and consumed
-across a disparate set of languages and societies, which would limit the
-applicability of the specification and significantly diminish its value as
-a standard.
+across a disparate set of languages and societies, which limits the
+applicability of the specification and significantly diminish its value as a
```
Comment (TallTed) [inline index.html]: `diminish` -> `diminishes`

File: index.html [hunk 3]
```diff
@@ -4854,8 +4854,8 @@ <h3>Language and Base Direction</h3>
       </pre>
 
       <p>
-An example utilizing the design pattern above is shown below to express the
-title of a book in the English language without specifying a text direction:
+Utilizing the design pattern above the following example expresses the title of
```
Comment (TallTed) [inline index.html]: `Utilizing the design pattern above the following`
-> `Using the design pattern above, the following`

File: index.html [hunk 4]
```diff
@@ -4907,30 +4907,36 @@ <h3>Language and Base Direction</h3>
...
+      </ul>
+
       <p>
-If implementers feel that they have to use HTML, or other markup languages
-that are capable of containing executable scripts, to achieve a particular
-use case, they are advised to analyze how an attacker would use the markup to
-mount injection attacks against a consumer of the markup and deploy mitigations
-for the identified attacks.
+If implementers feel they must use HTML, or other markup languages capable of
+containing executable scripts, to achieve a specific use case, they are advised
```
Comment (TallTed) [inline index.html]: `achieve` -> `address`

File: index.html (hunk 1 above)
Comment (grantnoble) [reply index.html]: Done.

File: index.html (hunk 2 above)
Comment (grantnoble) [reply index.html]: Done.

File: index.html (hunk 3 above)
Comment (grantnoble) [reply index.html]: Done.

File: index.html (hunk 4 above)
Comment (grantnoble) [reply index.html]: Done.

**Comment by grantnoble** [comment]
Thanks, @TallTed!

**Comment by grantnoble** [comment]
Updates completed.

**Pull request merged**